modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/mentions/strategy/PageEditorCommentMentionsStrategy.java

```diff
@@ -51,24 +49,17 @@ public List<User> getUsers(
 			JSONObject jsonObject)
 		throws PortalException {
 
-		SocialInteractionsConfiguration socialInteractionsConfiguration =
-			SocialInteractionsConfigurationUtil.
-				getSocialInteractionsConfiguration(
-					companyId, MentionsPortletKeys.MENTIONS);
-
-		List<User> users = _mentionsUserFinder.getUsers(
-			companyId, groupId, userId, query, socialInteractionsConfiguration);
-
-		long plid = jsonObject.getLong("plid");
```

Comment:
@JanisZhang Please keep this variable outside the ListUtil.filter lambda, to avoid repeatedly fetching this from the JSONObject (a map actually).